tasks/main.yml

```diff
@@ -172,3 +172,15 @@
       fail_on_error: yes
       strict: yes
   with_items: "{{ k8s_templates }}"
+
+- name: Do a rollout so we get our services updated even if image tags didn't change
```

Comment:
If a specific tag was used in the deploy, will this task trigger a 2nd restart of all the pods? (Does that matter?)